.gitlab-ci.yml

```diff
@@ -318,6 +321,11 @@ deploy-to-prod:
                 secretRef: mysql-password
               - name: OPENID_CONNECT_PARAMS
                 secretRef: openid-connect-params
+            - image: acreccuksprod.azurecr.io/portal-nginx-redirect:${CI_COMMIT_REF_SLUG}
```

Comment:
This block would deploy the redirect app as a container in the secure backend, you'll need another revision copy targeted at the new ACA that the terraform deploys. It's also worth mentioning that the only reason this uses YAML is that it contains multiple containers in one app, as the new one uses just one container in the app it may be possible to use just an `az containerapp revision copy` command with arguments to update the image.